app/javascript/hyacinth_v1/components/dynamic_fields/DynamicFieldForm.js

```diff
@@ -139,24 +139,32 @@ function DynamicFieldForm(props) {
 
       <Collapse in={fieldType === 'controlled_term'}>
         <div>
```

Comment:
Could these be rewritten to put the fieldType check at the top - basically to pull that internal `fieldType === '...'` logic outside the `<Collapse ...` and just set in=true?

Reply:
Good suggestion.  I just pushed a commit that addressed this, and a couple of other similar cases in the file.